chainer_compiler/elichika/parser/functions_builtin.py

```diff
@@ -47,22 +50,37 @@ def __init__(self):
         super().__init__()
         self.name = 'range'
 
-    def vcall(self, module: 'Field', graph: 'Graph', inst: 'values.ValueRef', args: 'functions.FunctionArgInput', line=-1):
-        node = nodes.NodeGenerate(
-            'range', [v.get_value() for v in args.inputs], line)
-        graph.add_node(node)
-        value = values.RangeValue()
-        value.name = '@F.{}.{}'.format(line, self.name)
-        node.set_outputs([value])
-        return values.ValueRef(value)
+    def vcall(self, module: 'values.Field', graph: 'graphs.Graph', inst: 'values.ValueRef', args: 'functions.FunctionArgInput',
+              option: 'vevaluator.VEvalOption' = None, line=-1):
+
+        if option._for_unroll:
+            for ref in args.inputs:
+                if not ref.get_value().has_constant_value():
+                    assert False
```

Comment:
How about `assert False, 'Loop unrolling was requested for non-constant sequence at %s' % lineinfo`? (If it's not easy to obtain line info here, you can skip it for now)

Reply:
Yeah, I guess that is a good idea.